db/migrate/20240719030053_remove_person_from_form_answers.rb

```diff
@@ -0,0 +1,6 @@
+class RemovePersonFromFormAnswers < ActiveRecord::Migration[7.1]
+  def change
+    remove_index :form_answers, :person_id
+    remove_column :form_answers, :person_id
+  end
+end
```

Comment:
So, if you remove `AddPersonToFormAnswers` migration you don't need this migration, right?

Reply:
Right! I'm just hesitant to alter existing migration files manually. But in this case, they were unneeded